easybuild/easyconfigs/s/SCENIC/SCENIC-1.3.0-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'RPackage'
+
+name = 'SCENIC'
+# local_commit = '0585e87'
```

Comment:
@lexming this doesn't appear to be doing anything?

Reply:
very true, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/19313/commits/b610cda136814753c5d5a37079fc41b2aae9a917